pkg/proc/gdbserial/gdbserver.go

```diff
@@ -1742,157 +1780,369 @@ func (regs *gdbRegisters) byName(name string) uint64 {
 }
 
 func (regs *gdbRegisters) Get(n int) (uint64, error) {
-	reg := x86asm.Reg(n)
 	const (
-		mask8  = 0x000f
-		mask16 = 0x00ff
-		mask32 = 0xffff
+		mask8  = 0xff
```

Comment:
This has been fixed elsewhere, but we really should export these somewhere for consistency.

Reply:
To be clear, I don't think this should happen in this patch, just making a note.

Reply:
It makes more sense to do it after the compositeMemory.WriteMemory PR, since that changes some other things about registers, we should just go through op.DwarfRegisters.